docs/docker-image-details.md

```diff
@@ -14,7 +14,7 @@ You can deploy the service with the Gravitino Docker image.
 Container startup commands
 
 ```shell
-docker run --rm -d -p 8090:8090 -p 9001:9001 apache/gravitino:0.6.0-incubating
+docker run --rm -d -p 8090:8090 -p 9001:9001 apache/gravitino:0.6.1-incubating
```

Comment:
Another place also need to be changed 
<img width="872" alt="image" src="https://github.com/user-attachments/assets/2ba21296-44cd-44ba-9546-0bf531c5789b">

Reply:
OK.